core/space/services/services_sharing.go

```diff
@@ -338,38 +353,38 @@ func (s *Space) HandleSharedFilesInvitation(ctx context.Context, invitationId st
 
 	if accept {
 		invitation, err = s.tc.AcceptSharedFilesInvitation(ctx, invitation)
-	} else {
-		invitation, err = s.tc.RejectSharedFilesInvitation(ctx, invitation)
-	}
-	if err != nil {
-		return err
-	}
 
-	// notify inviter,  it was accepted
-	invitersPk, err := decodePublicKey(err, invitation.InviterPublicKey)
-	if err != nil {
-		log.Error("should not happen, but inviters public key is invalid", err)
-		return errFailedToNotifyInviter
-	}
+		// notify inviter,  it was accepted
+		invitersPk, err := decodePublicKey(err, invitation.InviterPublicKey)
+		if err != nil {
+			log.Error("should not happen, but inviters public key is invalid", err)
+			return errFailedToNotifyInviter
+		}
 
-	messageBody, err := json.Marshal(&invitation)
-	if err != nil {
-		log.Error("error encoding invitation response body", err)
-		return errFailedToNotifyInviter
-	}
+		messageBody, err := json.Marshal(&invitation)
+		if err != nil {
+			log.Error("error encoding invitation response body", err)
+			return errFailedToNotifyInviter
+		}
 
-	message, err := json.Marshal(&domain.MessageBody{
-		Type: domain.INVITATION_REPLY,
-		Body: messageBody,
-	})
+		message, err := json.Marshal(&domain.MessageBody{
+			Type: domain.INVITATION_REPLY,
+			Body: messageBody,
+		})
 
+		if err != nil {
+			log.Error("error encoding invitation response", err)
+			return errFailedToNotifyInviter
+		}
+
+		_, err = s.tc.SendMessage(ctx, invitersPk, message)
+	} else {
+		invitation, err = s.tc.RejectSharedFilesInvitation(ctx, invitation)
```

Comment:
I see we don't want to notify inviter that his invite was rejected?

Reply:
Yeah I was thinking like when we get Github invites or docs I don't remember usually getting invites for rejections.  But spoke with Harrison on this and we can leave it like this and see if we want reject notifications when we do the polishing and tweak since it's a small change to do it on rejects as well.